src/Home.svelte

```diff
@@ -0,0 +1,73 @@
+<script lang="ts">
+    import Sidebar from './Sidebar.svelte';
+    import Upload from './Upload.svelte';
+    import {currentTemplate, template} from "./store";
+    import {createPop} from "./common";
+
+
+    let map;
+    fetch('http://localhost:19159/api/getconfig',{

```

Comment:
```suggestion
    fetch('/api/getconfig',{

```
写成这样应该外网就能访问了, 可以用process.env来区分开发和生产环境设置不同的地址